bigbluebutton-html5/imports/api/log-client/server/methods/logClient.js

```diff
@@ -1,10 +1,24 @@
 import Logger from '/imports/startup/server/logger';
+import Users from '/imports/api/users';
 
 const logClient = (type, log, ...args) => {
+  const SERVER_CONN_ID = Object.keys(this.Meteor.default_server.sessions)[0];
```

Comment:
When I run this inside a method, I can confirm that it shows the connection id. I am not sure why the structure is the way it is, do you know why?  {connection id: Session{...}}. Basically, I want to make sure we are picking the correct connection id. If I have 2 users in the meeting I see 2x connectionId: Session{...}. How do we know the one we pick is the right one?

Reply:
https://github.com/bigbluebutton/bigbluebutton/blob/v2.0.x-release/bigbluebutton-html5/imports/api/users/server/methods/validateAuthToken.js#L32  -- digging deeper - looks like we have access to `this.connection.id` when we are in the context of the method. Here it's a bit more complex because we have the ACL layer wrapping up.  I think we should look further into exposing `this.connection.id` here or alternatively, pass it when wrapping in ACL.

Reply:
This is plenty wrong. This connection id is by no means the connection of the client logging the message.

Reply:
You need to get the connection id from the method itself as @antobinary mentioned above.

Reply:
And please handle the cases where you dont find a user with that connection id. That can happen when loggin stuff when a meeting ends or the user disconnect.

Reply:
Ok